invisible_cities/cities/detsim_simulate_photons_test.py

```diff
@@ -0,0 +1,63 @@
+import os
+import numpy  as np
+import tables as tb
+import pandas as pd
+from pytest import fixture
+
+import invisible_cities.database.load_db as db
+
+from invisible_cities.io.mcinfo_io import read_mchits_df
+
+from invisible_cities.cities.detsim_simulate_photons import *
+
+## same as in detsim_simulate_electrons_test
+@fixture(params = ["Kr83_nexus_v5_03_00_ACTIVE_7bar_3evts.MCRD.h5"])
+def nexusMC_filename(request, ICDATADIR):
+    return os.path.join(ICDATADIR, request.param)
+
+## same as in detsim_simulate_electrons_test
+@fixture
+def random_event_hits(nexusMC_filename):
+    filename = nexusMC_filename
+
+    with tb.open_file(filename) as h5in:
+        extents = pd.read_hdf(filename, 'MC/extents')
+        event_ids  = extents.evt_number
+        hits_df    = read_mchits_df(h5in, extents)
```

Comment:
We'll have to change this with the new readers implementation so it works for the new format too. It's in PR #693 if you want to have a look, hopefully it'll be ready soon